src/libraries/System.Private.CoreLib/src/System/Threading/ThreadLocal.cs

```diff
@@ -27,6 +27,8 @@ namespace System.Threading
     [DebuggerDisplay("IsValueCreated={IsValueCreated}, Value={ValueForDebugDisplay}, Count={ValuesCountForDebugDisplay}")]
     public class ThreadLocal<T> : IDisposable
     {
+        private const int valuePerSlot = 64;
```

Comment:
```suggestion
        private const int ValuePerSlot = 64;
```
https://github.com/dotnet/runtime/blob/master/docs/coding-guidelines/coding-style.md